resources/views/blog/index.blade.php

```diff
@@ -0,0 +1,69 @@
+<?php ?>
+@extends('layouts.app')
+@section('content')
+
+<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#createModal">Create Blog</button>
```

Comment:
Where is the usage of permission for create ?

Reply:
Dạ đúng em dùng modal của bootstrap: https://getbootstrap.com/docs/4.0/components/modal/

Reply:
I fixed, Em đã check và thêm điều kiện vào blog controller